packages/asc-core/src/utils/themeUtils.ts

```diff
@@ -3,6 +3,15 @@ import { css } from '../styled-components'
 import { Theme } from '../theme'
 import { fromTheme } from '.'
 
+export const color = (
+  colorType?: Theme.TypeLevel,
+  variant: string = 'main',
```

Comment:
we can type the variants, as we know which ones are restricted (cannot be only just any string). perhaps we can use keyof here?

Reply:
will be done in the second iteration